data/org.freedesktop.impl.portal.GlobalShortcuts.xml

```diff
@@ -106,6 +106,28 @@
       <arg type="a{sv}" name="results" direction="out"/>
```

Comment:
At the top of this and the other xml files, it still says "this documentation describes version 1 of this interface"